src/test/java/com/google/api/codegen/testdata/discoveries/sheets.v4.json.overrides

```diff
@@ -1,12 +1,267 @@
 {
   "java": {
     "methods": {
+      "sheets.spreadsheets.batchUpdate": {
+        "requestBodyType": {
```

Comment:
Sai and I talked last week and decided to do the quick approach for now until we get an idea of how prevalent the issue is. Let's stick with that.

You're right that we should be careful in communications. The way I see that part working is that we'll try to generate samples for a team, they'll fail, we'll see this is needed, we'll ask the team to provide this, and we'll say we're working to simplify this mechanism if it turns out to be common.